api/src/Responder/Response/ApiResponse.php

```diff
@@ -20,6 +21,15 @@ class ApiResponse extends Response
     /** @var TransformerInterface */
     protected $transformer = null;
 
+    private $localizeToRegion = false;
+
+    public function __construct($localizeToRegion = false)
+    {
+        $this->localizeToRegion = $localizeToRegion;
+
+        parent::__construct();
```

Comment:
@jeremysik can you refactor this out so that there isn't an override of the parent constructor, as with this implementation the constructor interface has changed from 
`public function __construct($content = '', $status = 200, $headers = array())`

to

`public function __construct($localizeToRegion = false)`

which is unexpected, and the calling function might not know that the instance of Symfony\Component\HttpFoundation\Response that it has has a completely different constructor.

Instead, create a method `ApiResponse->setLocalizationRegion($region)` and immediately call it on the implementing controller eg:
```
    public function getResponse()
    {
        if ($region = RequestFacade::header('Accept-Region')) {
            return (new ApiResponse())->setLocalizationRegion($region);
        }

        return new ApiResponse();
    }
```